Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by build [OMERO-DEV-latest-docs-autogen#343](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/343/).

---

--no-rebase

hflynn commented:
Version history looks fine on staging. @jburel - okay to merge these time stamp changes too or would you prefer I do a manual PR to just update the history file?

jburel commented:
The time stamp is updated every time, we will need to look at that.
We have the auto-generated with that change,  I will keep it as it is

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#346 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/346/)

hflynn commented:
Updated history is fine on staging, merging for release.
